README.rst

```diff
@@ -69,9 +69,7 @@ All required dependencies are installed automatically when you install ``autorej
 
 Optional dependencies are:
 
-* ``tqdm`` (for nice progress-bars when setting ``verbose=True``)
-* ``h5io`` (for writing ``autoreject`` objects using the HDF5 format)
```

Comment:
tqdm and h5io are now part of mne deps, if we require mne >= 1.0 and mne[hdf5], ... that's why we can remove this here IMHO